### PR TITLE
Adding docs/extras/contribs.rst

### PR DESCRIPTION
**Pull request by Lucas-C**
_cf._ discussion in https://github.com/posativ/isso/issues/634

### CODE REVIEW

**Comment by Lucas-C** [comment]
Do you have any need for changes on this PR ? :)

File: docs/docs/extras/contribs.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Community tools
+===============
+
+Utility scripts
+---------------
+
+Some utility scripts have been developped by isso users:
```
Comment (jelmer) [inline docs/docs/extras/contribs.rst:7]: developed rather than developped

Could you add a brief overview of the scripts, i.e. a single line per script?

**Comment by Lucas-C** [comment]
PR updated with the suggested improvements :)

Preview on GitHub : https://github.com/Lucas-C/isso/blob/add-docs-contribs/docs/docs/extras/contribs.rst

**Comment by Lucas-C** [comment]
Thanks for merging this.
At else should be done to update the online docs at https://posativ.org/isso/docs/ ?
It doesn't seem hosted as GitHub Pages.

**Comment by jelmer** [comment]
On Thu, Apr 23, 2020 at 05:47:25AM -0700, Lucas Cimon wrote:
> Thanks for merging this.
> At else should be done to update the online docs at https://posativ.org/isso/docs/ ?
> It doesn't seem hosted as GitHub Pages.

I believe posativ@ hosts it; we'll need to get him to regenerate and
publish it at some point.

-- 
Jelmer Vernooĳ <jelmer@jelmer.uk>
PGP Key: https://www.jelmer.uk/D729A457.asc

**Pull request merged**